src/borg/platform/base.py

```diff
@@ -94,6 +94,8 @@ def get_flags(path, st, fd=None):
 
 
 def sync_dir(path):
+    if os.name == 'nt':
+        return
```

Comment:
there should be a comment here, clarifying this:
- is there no way to sync a directory on windows?
- or is this just a temp hack to get it working?

Reply:
Opening directories with `os.open` is not possible on windows. I don't know if it is possible or neccessary on windows to sync directories. For the moment I added a comment and a TODO.